src/handlers/auth.rs

```diff
@@ -109,12 +115,39 @@ pub fn auth(ctx_handle: &ContextHandle) -> HandlerResult {
     }
 
     let nonce = try_get_input_param!(params, "nonce");
-    let login_hint = try_get_input_param!(params, "login_hint");
     if try_get_input_param!(params, "response_type") != "id_token" {
         return Box::new(future::err(BrokerError::Input(
             "unsupported response_type, only id_token is supported".to_owned())));
     }
 
+    let login_hint = try_get_input_param!(params, "login_hint", "".to_string());
+    if login_hint == "" {
+        let data = mustache::MapBuilder::new()
+            // TODO: catalog/localization?
```

Comment:
Is this something you'd like to try, or should we take a look at it?

Reply:
I went ahead and implemented it, following the pattern of other localized strings.

I'm not qualified to localize for other languages, though, unfortunately.

Tests are green as well. :slightly_smiling_face: